src/twisted/trial/test/test_skipif.py

```diff
@@ -61,32 +63,3 @@ def test_shouldShouldRunWithSkipIfFalse(self):
     @skipIf(False, "")
     def test_shouldShouldRunWithSkipIfFalseEmptyReason(self):
         self.assertTrue(True, "Test should run and not be skipped")
-
-
-class SkipAttributeOnClass(TestCase):
-    """
-    All tests should be skipped because skip attribute is set on
-    this class.
-    """
-
-    skip = "'skip' attribute set on this class, so skip all tests"
```

Comment:
Where did test coverage of handling skip set on a class object go?

Reply:
Good question. In skipping.py there is SkippedClassMixin  which is used in SynchronousSkippedClass and  AsynchronousSkippedClass... which are used in test_tests.py in SynchronousSkipClassTests and AsynchronousSkipClassTests with assertions in SkipClassesMixin  ...

I tried to keep  this PR as small as possible but I can also add docstrings to SkippedClassMixin in skipping.py